instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -293,53 +303,51 @@ def custom_event_context_extractor(lambda_event):
             ),
         ]
         for test in tests:
+            with self.subTest(test_name=test.name):
+                test_env_patch = mock.patch.dict(
+                    "os.environ",
+                    {
+                        **os.environ,
+                        # NOT Active Tracing
+                        _X_AMZN_TRACE_ID: test.xray_traceid,
+                        OTEL_PROPAGATORS: test.propagators,
+                    },
+                )
+                test_env_patch.start()
+                reload(propagate)
 
-            test_env_patch = mock.patch.dict(
-                "os.environ",
-                {
-                    **os.environ,
-                    # NOT Active Tracing
-                    _X_AMZN_TRACE_ID: test.xray_traceid,
-                    OTEL_PROPAGATORS: test.propagators,
-                },
-            )
-            test_env_patch.start()
-            reload(propagate)
-
-            AwsLambdaInstrumentor().instrument(
-                event_context_extractor=test.custom_extractor,
-            )
-            result = mock_execute_lambda(test.context)
-            result = json.loads(result)
-
-            spans = self.memory_exporter.get_finished_spans()
-            assert spans
-            self.assertEqual(len(spans), 1)
-            span = spans[0]
-            self.assertEqual(
-                span.get_span_context().trace_id, test.expected_traceid
-            )
+                AwsLambdaInstrumentor().instrument(
+                    event_context_extractor=test.custom_extractor,
+                )
+                result = mock_execute_lambda(test.context)
+                json.loads(result)
```

Comment:
It looks like this was originally added by https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2589

I removed the reassignment of `result` since it was not used downstream.

Looking at the test case, it doesn't appear to be required, so has now been removed 😄